private/buf/bufcli/bufcli.go

```diff
@@ -802,30 +801,6 @@ func NewImageForSource(
 	return bufimage.MergeImages(images...)
 }
 
-// ParseInputAndType returns the moduleReference and typeName from the input and type provided by the user.
-// When input is not provided, we assume the type is a fully qualified path to the type and try to parse it.
-// Otherwise, if both input and type are provided, the type must be a valid Protobuf identifier (e.g. weather.v1.Units).
-func ParseInputAndType(
-	ctx context.Context,
-	input string,
-	typeName string,
-) (string, string, error) {
-	if input != "" && typeName != "" {
-		if err := bufreflect.ValidateTypeName(typeName); err != nil {
-			return "", "", err
-		}
-		return input, typeName, nil
-	}
-	if typeName == "" {
-		return "", "", appcmd.NewInvalidArgumentError("type is required")
-	}
-	moduleReference, moduleTypeName, err := parseFullyQualifiedPath(typeName)
-	if err != nil {
-		return "", "", appcmd.NewInvalidArgumentErrorf("if a input isn't provided, the type needs to be a fully qualified path that includes the module reference; failed to parse the type: %v", err)
-	}
-	return moduleReference, moduleTypeName, nil
-}
-
```

Comment:
This had to do with parsing the fully qualified type names `buf.build/org/name#typename`